ranger-drove/src/main/java/io/appform/ranger/drove/common/DroveCommunicator.java

```diff
@@ -30,6 +30,8 @@
 public interface DroveCommunicator extends AutoCloseable {
```

Comment:
Related to my comment on the HttpCommunicator - Increasingly this one now looks like a functional interface, with drove, http etc being implementations of it, and we can possibly do away with SDS in the future.

Reply:
will think about refactoring later on.